questionpy/form/_dsl.py

```diff
@@ -268,7 +393,12 @@ def _condition_with_value_factory(model: Type[_C]) -> Callable[[str, Any], _C]:
 
 
 is_checked = _condition_factory(IsChecked)
+"""Condition on a checkbox being checked."""
 is_not_checked = _condition_factory(IsNotChecked)
+"""Condition on a checkbox being unchecked."""
 equals = _condition_with_value_factory(Equals)
+"""Condition on the value of another field being equal to some static value."""
 does_not_equal = _condition_with_value_factory(DoesNotEqual)
+"""Condition on the value of another field not being equal to some static value."""
 is_in = _condition_with_value_factory(In)
+"""Condition on the value of another field being one of a number of static values."""
```

Comment:
Bitte beschreibe diese Funktionen deutlich ausführlicher.
Durch die Verwendung der Factories ist es ziemlich schwierig zu verstehen, wie diese Funktionen benutzt werden können. Die IDE kann einem auch nicht helfen, welche Argumente beispielsweise `is_in` benötigt.

Vielleicht wäre es für Anwender auch besser, wenn wir die Funktionen "ausschreiben", damit die Argumente klar verständlich sind.
Die Values für equals/does_not_equal/is_in könnten wir bei der Gelegenheit auch auf str, bool und int festsetzen. Floats sollte man nicht einfach so miteinander vergleichen und andere in Formularen passende Datentypen fallen mir gerade nicht.